modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -561,6 +561,14 @@ public ObjectRelationship getObjectRelationship(
 		}
 	}
 
+	@Override
+	public ObjectRelationship getObjectRelationshipByExternalReferenceCode(
```

Comment:
I think we should rename this method for `fetchObjectRelationshipByExternalReferenceCode()` if we call the method `fetch` in the persistence class.